src/App.js

```diff
@@ -1,43 +1,47 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import WatchLater from "./pages/WatchLater";
+import History from "./pages/History";
+import MyPlaylists from "./pages/MyPlaylists";
```

Comment:
you can have one index.js file in pages folder and import all these components there and then export, it will reduce the number of imports that you have to write on this page.

Reply:
Nice suggestion will do in the upcoming PRs